tutorcairn/patches/k8s-volumes

```diff
@@ -44,3 +44,17 @@ spec:
     requests:
       storage: 2Gi
 {% endif %}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cairn-watchcourses
+  labels:
+    app.kubernetes.io/component: volume
+    app.kubernetes.io/name: cairn-watchcourses
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Why are we creating a PVC here? This PVC is unused, unless I'm mistaken.

Reply:
You are correct, this PVC is unused. I will remove it.